### PR TITLE
feat(timeline): init component

### PR DESCRIPTION
**Pull request by degliwe**

### CODE REVIEW

**Comment by yhuard** [comment]
Starting the QA

**Comment by yhuard** [comment]
Ok, component moved into molecules. Empty JS script added. Ready for QA second pass.

File: framework/molecules/timelines/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Timeline
```
Comment (kalinchernev) [inline framework/molecules/timelines/README.md]: => `Timelines` ;)

File: framework/molecules/timelines/timelines.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  title: 'Timelines',
+  label: 'Timelines',
+  status: 'wip',
```
Comment (kalinchernev) [inline framework/molecules/timelines/timelines.config.js]: I'd say it's ready

File: framework/molecules/timelines/timelines.config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  collated: true,
+  preview: '@preview-center-transparent',
+  context: {
+    _demo: {
+      scripts: 'Europa.timeline();',
```
Comment (kalinchernev) [inline framework/molecules/timelines/timelines.config.js]: I wouldn't include this at the moment as it's not working anyway.

File: framework/molecules/timelines/timelines.config.js (hunk 2 above)
Comment (yhuard) [inline framework/molecules/timelines/timelines.config.js]: It's not until the JS works

File: framework/molecules/timelines/README.md (hunk 1 above)
Comment (yhuard) [inline framework/molecules/timelines/README.md]: 👍

File: framework/molecules/timelines/timelines.config.js (hunk 3 above)
Comment (yhuard) [inline framework/molecules/timelines/timelines.config.js]: It's just for the demo, to show how to include a JS function call in a pen. I agree it does nothing interesting and is not mandatory, but it also reminds us that the JS behaviour is not implemented yet

**Pull request merged**